test_utils/utils.go

```diff
@@ -16,13 +21,81 @@ import (
 
 	"github.com/containernetworking/cni/pkg/ns"
 	"github.com/containernetworking/cni/pkg/types"
+	etcdclient "github.com/coreos/etcd/client"
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega/gexec"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"github.com/projectcalico/libcalico-go/lib/client"
+	calnet "github.com/projectcalico/libcalico-go/lib/net"
 	"github.com/vishvananda/netlink"
 )
 
-func PreCreatePool(pool string) string {
-	return Cmd("dist/calicoctl pool add " + pool)
+var kapi etcdclient.KeysAPI
+
+func init() {
+
+	cfg := etcdclient.Config{Endpoints: []string{"http://127.0.0.1:2379"}}
+	c, _ := etcdclient.New(cfg)
+	kapi = etcdclient.NewKeysAPI(c)
+}
+
+// Delete everything under /calico from etcd
```

Comment:
All nice function to have, but again - would be useful to have in our testutils :-)